### PR TITLE
diff : Add Korean Translation

### PR DESCRIPTION
**Pull request by Proscream**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ko/diff.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# diff
+
+> 파일들과 디렉토리들을 비교한다.
+
+- 파일들 비교하기 (lists changes to turn `이전_파일명`을 `새_파일명`으로 바꾸는 변경점들 목록):
```
Comment (zdroid) [inline pages.ko/diff.md]: This should be translated aswell, I guess.

**Comment by zdroid** [comment]
I honestly don't know what's causing the build error. Any ideas?

**Comment by sbrl** [comment]
Looks like it's an issue with the recent updates to the indexer, looking at the Travis CI output?

/cc @mebeim / @pxgamer

**Comment by pxgamer** [comment]
Looks like this is because it isn't in a platform directory. I think if the page is moved to `pages.ko/common`, it should be fine. :+1:

**Comment by zdroid** [comment]
Ohh right.

**Comment by mebeim** [comment]
@sbrl I'll try to inspect and take a look when I can.

**Comment by zdroid** [comment]
@mebeim @sbrl it isn't, check the latest commit.

**Comment by zdroid** [comment]
Whoops. I was blocking this unintentionally.

**Comment by sbrl** [comment]
Looks like the issue was identified by @pxgamer, @mebeim :-)

**Pull request merged**